backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Tests/Controllers/ResourceControllerTest.cs

```diff
@@ -94,18 +96,20 @@ public async Task GetSingleRightsSearch_searchStringAndFilterNotSet_ReturnsAll()
         }
 
         /// <summary>
-        /// Test case: PaginatedSearch with pagination and filters
-        /// Expected: PaginatedSearch returns a list of resources matching the filters in paginated form with language filtered for the authenticated users selected language
+        ///     Test case: PaginatedSearch with pagination and filters
+        ///     Expected: PaginatedSearch returns a list of resources matching the filters in paginated form with language filtered
+        ///     for the authenticated users selected language
         /// </summary>
         [Fact]
         public async Task GetSingleRightsSearch_filterSet_ReturnsMatch()
         {
             // Arrange
-            var token = PrincipalUtil.GetToken(1337, 501337, 2);
+            string token = PrincipalUtil.GetToken(1337, 501337);
             _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
             int page = 1;
             int resultsPerPage = 4;
-            string[] roFilters = { "777777777", "123456789" }; // Narnia and testdepartementet
+            // Narnia and testdepartementet
+            string[] roFilters = { "777777777", "123456789" };
```

Comment:
Not a major thing, but I usually prefer to keep the comments on the same line as the code in cases such as these, where the line is short enough to allow it, and the comment only clarifies the right side of equation.
If the comment is on its own line like this change introduces, it should probably be more descriptive of the entire line of code as a whole. Such as 'Set resource owner filters to narnia and testdepartementet'

Reply:
Why do you think it should be more descriptive, what else can Narnia and testdepartementet possibly mean? And where did you read about that sidecomments should be on the right side, I've never seen anything about it?

Reply:
Like I said, this is a very small thing, more of a personal preference.

I haven't read it anywhere, I just think it makes sense structurally.
Like the difference between a title over a paragraph and a parathesis at the end of a sentence.  In the first case, I expect the title to pertain to the whole paragraph, but in the second case it can just be some added information related to the last thing mentioned in the sentence.